common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2085,6 +2091,12 @@
     },
     "sqlState" : "22023"
   },
+  "INVALID_PARAMETRIZED_QUERY" : {
```

Comment:
This seems generic name but the error is pretty specific. Please, reformulate the error class name.

Reply:
No, we don't because there is not method in Scala API which accepts both named and positional parameters.

Reply:
Yes, scala API accepts Array and assumes positional arguments, or Map and assumes named parameters